apps/web/public/sitemap.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
+<url><loc>https://sseraphini.cc.com/</loc><changefreq>weekly</changefreq><priority>0.8</priority></url>
+<url><loc>https://sseraphini.cc.com/search</loc><changefreq>weekly</changefreq><priority>0.8</priority></url>
+<url><loc>https://sseraphini.cc.com/score</loc><changefreq>weekly</changefreq><priority>0.8</priority></url>
```

Comment:
How do we update this?

Reply:
for now, manually, adding new pages inside of this file, but we can do it programmatically as well, but I didn't see needing it for now, because there are few pages.

WDYT?